frontend/src/components/UserProfile.jsx

```diff
@@ -1,12 +1,70 @@
 import "./UserProfile.css";
+import { useContext } from "react";
+import {UserContext} from './InfoProvider';
```

Comment:
your imports should be consistent, so if there is an empty space around "useContext" there should be empty spaces around "UserContext" too, and the same consistently in every single file